fineract-provider/src/test/java/org/apache/fineract/notification/Listener.java

```diff
@@ -29,6 +29,6 @@
     @Override
     public void onMessage(Message message, Session session) throws JMSException {
         TextMessage msg = (TextMessage) message;
-        System.out.println("Received: " + msg.getText());
+        LOG.debug("Received: " + msg.getText());
```

Comment:
it's probably OK, and likely better/clearer, to simply always use `LOG.info()` instead of `debug()` to replace `System.out.println()` in tests. In "real" (non-test) code, it's... not always super obvious which level is right. But, hopefully, you won't find any `System.out.println()` in non-test code?

Reply:
Yes, no System.out  in a non-test folder.
Replacing all of them with LOG.info :)


-thanks 
Manthan